run-build.sh

```diff
@@ -79,19 +71,11 @@ else
     exit 1
 fi
 
-# Load Branch Info
-while read line; do
-    if [[ $line != \#* ]]; then
-        IFS='=' read -ra splat <<< "$line"
-        export ${splat[0]}="${splat[1]}"
-    fi
-done < "$REPOROOT/branchinfo.txt"
-
 # Use a repo-local install directory (but not the artifacts directory because that gets cleaned a lot
-[ -z "$DOTNET_INSTALL_DIR" ] && export DOTNET_INSTALL_DIR=$REPOROOT/.dotnet_stage0/$(uname)
+[ -z "$DOTNET_INSTALL_DIR" ] && export DOTNET_INSTALL_DIR=$REPOROOT/.dotnet_stage0/$ARCHITECTURE
 [ -d "$DOTNET_INSTALL_DIR" ] || mkdir -p $DOTNET_INSTALL_DIR
```

Comment:
Here's a case where the .sh script does something in one place, but the .ps1 scripts do it in another place.

We should align the two sets of scripts so the exact same steps occur in the exact same place in the corresponding script.